device/dhcp/clients/v4/dhcpv4client.yaml

```diff
@@ -0,0 +1,60 @@
+components:
+  schemas:
+    Device.Dhcpv4client:
+      description: >-
+        Configuration for emulated DHCPv4 Client on a single Interface.
+        https://www.rfc-editor.org/rfc/rfc2131.html
+      type: object
+      x-status:
+        status: under_review
+      required: [name]
+      properties:
+        name:
+          x-include: ../../../../common/common.yaml#/components/schemas/Named.Object/properties/name
+          x-field-uid: 1
+        choice:
```

Comment:
I feel `server_address` or `first_server` is a require field. So, choice will be require/ set `first_server`  as default.